src/API/useInAppDownloadPreview.js

```diff
@@ -5,11 +5,16 @@ import {
 import { useOkapiKy } from '@folio/stripes/core';
 
 export const useInAppDownloadPreview = (id, capability) => {
+  const capabilityMapping = {
```

Comment:
The same object we have in usePreviewRecords.js urlMapping object. Maybe we can move it to constants.js